src/landIce/problems/LandIce_StokesFOBase.cpp

```diff
@@ -120,7 +120,30 @@ void StokesFOBase::buildProblem (Teuchos::ArrayRCP<Teuchos::RCP<Albany::MeshSpec
   cellType = rcp(new shards::CellTopology (cell_top));
 
   Intrepid2::DefaultCubatureFactory cubFactory;
-  cellCubature = cubFactory.create<PHX::Device, RealType, RealType>(*cellType, meshSpecs[0]->cubatureDegree);
+
+  int defaultCubDegree = 3;
+  bool tensorProductCell = (cellType->getKey() == shards::Wedge<6>::key) || (cellType->getKey() == shards::Hexahedron<8>::key);
```

Comment:
Technically, a hexa may not be an extruded mesh. We need the mesh to be extruded, since we need the elements to always be oriented in the same way. I think stokesFO has access to the disc param list, so we could ping that.. Alternatively, we could add a bool flag in the mesh specs (whether the mesh is extruded or not, is a true global info that deserves to be in the MS).

Reply:
Oh right. We talked about this. I'll fix that.